server/utils/editAppointmentUtils.ts

```diff
@@ -139,27 +182,27 @@ const getEditHintAction = (appointmentJourney: AppointmentJourney, editAppointme
   return 'changing'
 }
 
-export const minAppointmentSequenceNumber = (editAppointmentJourney: EditAppointmentJourney) =>
-  Array.isArray(editAppointmentJourney.sequenceNumbers) && editAppointmentJourney.sequenceNumbers.length
-    ? Math.min(...editAppointmentJourney.sequenceNumbers)
-    : editAppointmentJourney.sequenceNumber
+export const getOrderedOccurrences = (editAppointmentJourney: EditAppointmentJourney) =>
+  editAppointmentJourney.occurrences.sort((a, b) => a.sequenceNumber - b.sequenceNumber)
```

Comment:
I'm not sure if sorting occurrences should be done here or when the session is set. If sorted when set in theory it should only need to be done once.

My concern was that the purpose of a sort in startJourney.ts wouldn't be clear and these util functions would then need to assume the array is pre-sorted.

Adding the sort here and calling it from other util methods seemed like the clearest and safest approach.